data/status/status_fuv.xml

```diff
@@ -15,6 +15,7 @@
 <bridgehead id='history'>History</bridgehead>
 
 <history>
+  <history-entry date='July 24, 2021'>OHCHR identifies the language as Peuhl and credits the translation to Ministère de l'Education Nationale, Guinea. Therefore, the language must be Pulaar (ISO 639-3 'fuc', not 'fuv'). Changed accordingly. Thanks to Adam Twardoch.</history-entry>
```

Comment:
Compare OHCHR fum and OHCHR fum1. Same text, but different language names and sources. I don't think we have enough evidence to change anything.